app/models/hmpps_api/offender_base.rb

```diff
@@ -195,6 +195,10 @@ def within_early_allocation_window?
 
   private
 
+    def ldu
+      @case_information&.local_delivery_unit || @case_information&.local_divisional_unit
```

Comment:
See previous comment ^^^ we could perform a check here and only use the new Local Delivery Unit if it's `.enabled?`

---

Also: it'd be nice to put a comment on this method highlighting that this is where the 'fallback' happens.

Without a comment, it could look confusing since the two model names are deceptively similar – at a skim, it might look like we're returning the same thing twice. Bit of a gotcha hiding in there.